templates/web/common/site_layout.tt.html

```diff
@@ -249,21 +249,9 @@ <h1 itemprop="name">[% title %]
 									[% content %]
 								</div>
 							[% ELSE %]
-								<div class="row">
```

Comment:
This should be kept, if there is no schema_org_itemtype defined. (today there's always the "Thing" schema by default, but that may change).